### PR TITLE
Fix default mod_sec values, add namespace tag.

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: charts/generic-service/values.yaml [hunk 1]
```diff
@@ -60,29 +60,28 @@ ingress:
...
-  #      pass,\
-  #      t:none,\
-  #      setvar:tx.paranoia_level=2"
+  modsecurity_enabled: false
+  modsecurity_audit_enabled: false
+  modsecurity_github_team: ""
+  modsecurity_snippet: |
+    SecRuleEngine On
+    {{ if .Values.ingress.modsecurity_audit_enabled -}}
+    SecAuditEngine On
```
Comment (mattops) [inline charts/generic-service/values.yaml:69]: This is now `On` was previously `RelevantOnly`

File: charts/generic-service/values.yaml [hunk 2]
```diff
@@ -60,29 +60,28 @@ ingress:
...
+    SecAuditLog /var/log/nginx/error.log
+    SecAuditLogType Serial
+    {{ end -}}
+    {{ if .Values.ingress.modsecurity_github_team -}}
+    SecDefaultAction "phase:2,pass,log,tag:github_team={{ .Values.ingress.modsecurity_github_team }},tag:github_team={{ .Release.Namespace }}"
```
Comment (mattops) [inline charts/generic-service/values.yaml:74]: added namespace tag here.

**Review by williamfalconeruk — APPROVED** [review]
Looks Good!

**Comment by williamfalconeruk** [comment]
one further comment - perhaps we need to add a comment line to explain the snippet is a default and would be overridden when specified on values files? what do you think @mattops ?

File: charts/generic-service/values.yaml (hunk 2 above)
Comment (petergphillips) [reply charts/generic-service/values.yaml:74]: 👀  I can't see a namespace tag here, only two `github_team` tags 😂

Comment (williamfalconeruk) [reply charts/generic-service/values.yaml:74]: oops good call!

Comment (williamfalconeruk) [inline charts/generic-service/values.yaml:74]:
```suggestion
    SecDefaultAction "phase:2,pass,log,tag:github_team={{ .Values.ingress.modsecurity_github_team }},tag:namespace={{ .Release.Namespace }}"
```

**Review by williamfalconeruk — APPROVED** [review]

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**